gulpfile.js

```diff
@@ -41,18 +39,7 @@ gulp.task("test", function () {
 		});
 });
 
-gulp.task("doc", function () {
```

Comment:
Don't take this out, still nice to have a single `.md` file with all the docs.
![giphy 2](https://cloud.githubusercontent.com/assets/6805534/17407274/9efe83d6-5a34-11e6-8c04-38ba809ef985.gif)